package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "dependencies": {
     "body-parser": "1.17.2",
+    "cors": "^2.8.4",
```

Comment:
@Gwillison415 please remove the `^` here to lock the dependency in place. (and if you don't mind, check out the other dependencies too).